library/examples/hate_speech_multi_hf.py

```diff
@@ -69,7 +69,7 @@
 
 
 huggingface_base_config = HuggingfaceConfig(
-    preferred_load_origin=LoadOrigin.local,
+    preferred_load_origin=None,  # LoadOrigin.local,
```

Comment:
is it allowed for this to be None? what would that mean? I guess even if we want to train, we need LoadOrigin.pretrained, no?

Reply:
alright! so this should remain `None`, except for? when would we ever set a load origin?

Reply:
If you want to specifically select a source to load from. eg.: you know you'd like to train a pretrained model or you'd like to retrain a model you have already trained. 

But initally this was all automatically handled: if there is a local version use that, if not check for it on remote, if that doesnt exist load a pretrained model.

So this is just another option to be able to overwrite where you are getting the model from.